src/main/java/com/northconcepts/datapipeline/foundations/examples/jdbc/GenerateSchemaDefFromJdbc.java

```diff
@@ -0,0 +1,29 @@
+package com.northconcepts.datapipeline.foundations.examples.jdbc;
+
+import com.northconcepts.datapipeline.foundations.jdbc.JdbcConnection;
+import com.northconcepts.datapipeline.foundations.schema.SchemaDef;
+import com.northconcepts.datapipeline.foundations.tools.GenerateSchemaFromJdbc;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class GenerateSchemaDefFromJdbc {
+
+    public static void main(String[] args) {
+        JdbcConnection jdbcConnection = new JdbcConnection()
+                .setDriverClassName("com.mysql.jdbc.Driver")
+                .setUrl("jdbc:mysql://127.0.0.1:3306")
+                .setUsername("username")
+                .setPlainTextPassword("password");
+
+        GenerateSchemaFromJdbc generator = new GenerateSchemaFromJdbc(jdbcConnection);
+        
+        List<String> tableNames = new ArrayList<>();
+        tableNames.add("tableName");
+        tableNames.add("tableName");
```

Comment:
Can we use `Arrays.asList("tableName1", "tableName2")` and inline it?

Reply:
Done